payments/models.py

```diff
@@ -270,7 +270,12 @@ def total_pretax_price(self):
     @property
     @rounded
     def total_tax_percentage(self):
-        return ((self.total_price - self.total_pretax_price) / self.total_price) * 100
+        return rounded_decimal(
```

Comment:
we probably don't need the decorator then?

Reply:
True, not needed indeed 🔥